flink-contrib/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackendTest.java

```diff
@@ -215,46 +215,6 @@ public Object answer(InvocationOnMock invocationOnMock) throws Throwable {
 	}
 
 	@Test
-	public void testRunningSnapshotAfterBackendClosed() throws Exception {
```

Comment:
This test doesn't make sense anymore because a running snapshot operation will keep the backend form closing?

Reply:
Exactly.